@xen-orchestra/lite/src/App/PoolTab/PoolUpdates.tsx

```diff
@@ -53,21 +53,17 @@ const PoolUpdates = withState<State, Props, Effects, Computed, ParentState, Pare
   {
     computed: {
       availableUpdates: async function (state, { hostRef }) {
-        const error = <IntlMessage id='errorOccurred' />
         try {
-          const stringifiedPoolUpdates = await state.xapi.call(
+          const stringifiedPoolUpdates = (await state.xapi.call(
             'host.call_plugin',
             hostRef,
             'updater.py',
             'check_update',
             {}
-          )
-          if (typeof stringifiedPoolUpdates !== 'string') {
-            return error
-          }
+          )) as string
           return JSON.parse(stringifiedPoolUpdates)
         } catch (err) {
-          return error
+          return <IntlMessage id='errorOccurred' />
```

Comment:
IMHO it's a bad idea for debugging to completely hide an error, logging it on the console would be better.

Reply:
Then let's log the error before returning.